Description: Fix test binary_output requiring compiled bin
 by effectively embedding it; what it does is so simple, yet has to be "found"
 by an entire crate. Also remove dependency on that crate.
Forwarded: not-needed
Last-Update: 2024-07-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/tests/binary.rs
+++ b/tests/binary.rs
@@ -1,4 +1,2 @@
-extern crate executable_path;
-
-use {executable_path::executable_path, std::process::Command};
+use std::process::Command;
 
@@ -50,7 +48,11 @@
 fn binary_output() {
-  let output = Command::new(executable_path("target"))
-    .output()
-    .unwrap()
-    .stdout;
-  let output = std::str::from_utf8(&output).unwrap();
+  let output = format!(r#"arch: {}
+os: {}
+family: {}
+env: {}
+endian: {}
+pointer_width: {}
+vendor: {}
+feature: {:?}
+"#, target::arch(), target::os(), target::family(), target::env(), target::endian(), target::pointer_width(), target::vendor(), target::features());
   pretty_assertions::assert_eq!(output, EXPECTED);
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -19,4 +19,2 @@
 license = "CC0-1.0"
-[dev-dependencies.executable-path]
-version = "1.0.0"
 
